Makefile

```diff
@@ -57,4 +57,28 @@ dist: clean
 	ls -l dist
 
 install: clean
-	python3 setup.py install
\ No newline at end of file
+	python3 setup.py install
+
+h := $(shell git rev-parse --abbrev-ref HEAD)
+
+release: dist
+	if [ "$h" = "develop" ];\
+	then\
+        echo "Please ensure everything is merged into master & tagged there";\
+        pip install twine;\
```

Comment:
why not adding `twine` to `Pipfile`?

Reply:
because this command is only done by me and so it seems unnecessary?